core/domain/feedback_services.py

```diff
@@ -724,7 +724,32 @@ def _get_all_recipient_ids(exploration_id, thread_id, author_id):
     return (batch_recipient_ids, other_recipient_ids)
 
 
-def _send_batch_emails(recipient_list, feedback_message_reference):
+def _can_receive_email(recipient_id, exploration_id, has_suggestion):
```

Comment:
Should I then use this function in feedback_services or handle all logic in email_manager?